openedx/features/assessment/helpers.py

```diff
@@ -37,7 +38,11 @@ def find_and_autoscore_submissions(enrollments, submission_uuids):
     Returns:
         None
     """
-    days_to_wait = configuration_helpers.get_value('DAYS_TO_WAIT_AUTO_ASSESSMENT', DAYS_TO_WAIT_AUTO_ASSESSMENT)
+    days_to_wait = get_config_value_from_site_or_settings('DAYS_TO_WAIT_AUTO_ASSESSMENT', get_site(site_id))
```

Comment:
Can we use a Constant here to for this key 'DAYS_TO_WAIT_AUTO_ASSESSMENT' ?

Reply:
Considering the following points I think we should not make it constant:

1. `DAYS_TO_WAIT_AUTO_ASSESSMENT ` itself is a constant see `edx-platform/openedx/features/assessment/constants.py:4`
2. It has only one occurrence
3. In project we have used `configuration_helpers.get_value()` everywhere in same pattren

Reply:
yes you are right. Thank for your clarification